packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/DeviceOrientationManager.java

```diff
@@ -124,8 +123,8 @@ static void handleOrientationChange(
    */
   // Configuration.ORIENTATION_SQUARE is deprecated.
   @SuppressWarnings("deprecation")
-  @VisibleForTesting
```

Comment:
Why was `@VisibleForTesting` removed? It looks like it's still accessed from the same package, right?

Reply:
I'm using it beyond a test now so the linter through an error :/